lintconf.yaml

```diff
@@ -0,0 +1,42 @@
+---
+rules:
```

Comment:
What's this? Same as above, if it's really needed, can it be moved into the contrib folder, or is it useful for the entire project?